src/Security/Security/Cmdlets/SqlVulnerabilityAssessment/AddSqlVulnerabilityAssessmentBaseline.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Commands.Security;
+using Microsoft.Azure.Commands.Security.Common;
+using Microsoft.Azure.Commands.SecurityCenter.Models.SqlVulnerabilityAssessment;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.SecurityCenter.Cmdlets.SqlVulnerabilityAssessment
+{
+    [Cmdlet(VerbsCommon.Add, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlVulnerabilityAssessmentBaseline"),OutputType(typeof(PSSqlVulnerabilityAssessmentBaselineResults))]
```

Comment:
Add shouldprocess, please refer to https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/should-process-confirm-impact.md